app/views/posts/index.html.erb

```diff
@@ -26,7 +26,4 @@
 
 <%= link_to 'New Post', new_post_path %>
 
-<!--
-<% if current_user && current_user.editor? %> 
```

Comment:
editor na razie nie potrzebny, nie ma weryfikacji roli (roli też te nie ma)
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Tuesday, January 26, 2016 3:19 PM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In app/views/posts/index.html.erb:

> @@ -26,7 +26,4 @@
>  
>  <%= link_to 'New Post', new_post_path %>
>  
> -<!--
> -<% if current_user && current_user.editor? %> 
do odkomentowania, znowu editor - czy potrzebny?

  —
  Reply to this email directly or view it on GitHub.